recipes-domx/meta-xt-prod-vgpu/recipes-graphics/gles-module/gles-user-module_1.7.bbappend

```diff
@@ -1 +0,0 @@
-DEPENDS += " gles-module-egl-headers"
```

Comment:
put a comment on the patch description on the reasons for this file removal, e.g. only 1.9 is used?

Reply:
Done in commit msg